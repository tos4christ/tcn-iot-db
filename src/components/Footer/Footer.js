import React from "react";
const style = {
  marginTop: "90px",
  marginBottom: "-50px",
};
const Footer = () => {
  return (
    <div style={style} className="copyright-area">
      <p>
        Copyright &copy;
        {new Date().getFullYear()} | This web app is made with{" "}
        <i className="fa fa-heart-o" aria-hidden="true"></i> by Team-003
      </p>
    </div>
  );
};

export default Footer;
