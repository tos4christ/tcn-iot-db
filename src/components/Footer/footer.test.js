import React from "react";
import renderer from 'react-test-renderer';
import "@testing-library/jest-dom/extend-expect";
import Footer from "./Footer";


it("renders footer component", () => {
  let tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot()
});
