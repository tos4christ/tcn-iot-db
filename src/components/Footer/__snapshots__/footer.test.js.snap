// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer component 1`] = `
<div
  className="copyright-area"
  style={
    Object {
      "marginBottom": "-50px",
      "marginTop": "90px",
    }
  }
>
  <p>
    Copyright Â©
    2020
     | This web app is made with
     
    <i
      aria-hidden="true"
      className="fa fa-heart-o"
    />
     by Team-003
  </p>
</div>
`;
